composer/lib/dependabot/composer/file_fetcher.rb

```diff
@@ -90,17 +126,24 @@ def path_dependencies
           end
       end
 
+      def artifact_sources
+        sources.select { |details| details["type"] == "artifact" }.map { |details| details["url"] }
+      end
+
       def path_sources
-        @path_sources ||=
+        sources.select { |details| details["type"] == "path" }.map { |details| details["url"] }
+      end
+
+      def sources
+        @sources ||=
           begin
             repos = parsed_composer_json.fetch("repositories", [])
             if repos.is_a?(Hash) || repos.is_a?(Array)
               repos = repos.values if repos.is_a?(Hash)
               repos = repos.select { |r| r.is_a?(Hash) }
 
               repos
-                .select { |details| details["type"] == "path" }
-                .map { |details| details["url"] }
+                .select { |details| details["type"] == "path" || details["type"] == "artifact" }
```

Comment:
seems like the `artifact_sources` method?

Reply:
It is, but this method is memoized so we don't have to parse the composer.json twice. 

There might be a better way to do this but this seemed ok.